app/Http/Controllers/Chat/ChatController.php

```diff
@@ -313,22 +313,7 @@ public function updates()
         }
 
         if ($includeSilences) {
-            $silenceQuery = UserAccountHistory::bans()->limit(100);
-            $lastHistoryId = $params['history_since'];
-
-            if ($lastHistoryId === null) {
-                $previousMessage = Message::where('message_id', '<=', $since)->last();
-
-                if ($previousMessage === null) {
-                    $silenceQuery->none();
-                } else {
-                    $silenceQuery->where('timestamp', '>', $previousMessage->timestamp);
-                }
-            } else {
-                $silenceQuery->where('ban_id', '>', $lastHistoryId)->reorderBy('ban_id', 'DESC');
-            }
-
-            $silences = $silenceQuery->get();
+            $silences = $this->getSilences($params['history_since'], $params['since']);
```

Comment:
there's already `$since` which only used once after this change. Maybe use either and not both